pulp_2to3_migration/app/plugin/rpm/migrator.py

```diff
@@ -60,5 +87,95 @@ class RpmMigrator(Pulp2to3PluginMigrator):
         Migrate pre-migrated Pulp 2 RPM plugin content.
         """
         first_stage = ContentMigrationFirstStage(cls)
-        dm = DeclarativeContentMigration(first_stage=first_stage)
+        dm = RPMDeclarativeContentMigration(first_stage=first_stage)
         await dm.create()
+
+
+class RPMDeclarativeContentMigration(DeclarativeContentMigration):
+    """
+    A pipeline that migrates pre-migrated Pulp 2 rpm content into Pulp 3.
+    """
+
+    def pipeline_stages(self):
+        """
+        Build a list of stages.
+
+        This defines the "architecture" of the content migration to Pulp 3.
+
+        Returns:
+            list: List of :class:`~pulpcore.plugin.stages.Stage` instances
+
+        """
+        pipeline = [
+            self.first_stage,
+            QueryExistingArtifacts(),
+            ArtifactSaver(),
+            QueryExistingContents(),
+            ContentSaver(),
```

Comment:
I think we still need a RemoteArtifactStage and ContentSaver one. Or did you exclude it on purpose?
https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/plugin/content.py#L70

Reply:
contentsaver is there, i forgot remoteartifact saver, thanks for pointing out

Reply:
Ah, I meant `after` ContentSaver, sorry. Thanks!